python/gps/algorithm/policy_opt/policy_opt_tf.py

```diff
@@ -200,19 +240,70 @@ def prob(self, obs):
 
         return output, pol_sigma, pol_prec, pol_det_sigma
 
+    def linearize(self, obs):
```

Comment:
We can keep this, but I'm not positive it's correct, and it's not used anywhere else. I had some weird issues when I was trying to use analytic linearizations rather than the policy prior GMM.